ippserver/behaviour.py

```diff
@@ -48,13 +48,21 @@ def prepare_environment(ipp_request):
     return env
 
 
+def env(var):
+    """Get (bytes) value from environment"""
+    value = os.environ.get(var)
```

Comment:
I don't like the mix of `argparse` and `os.environ`, as it gives multiple ways to configure things at the same time.

But I can't think of another way to do this. (I think the "argparse kool aid" way would be to have a classmethod on the Behaviour which returns the subparser it would like? That feels over-complicated.)

So there's no good reason not do do it like this. Especially as the impact of forgetting to set `IPP_PRINTER_NAME` is very low.